.golangci.yml

```diff
@@ -0,0 +1,17 @@
+linters:
```

Comment:
Nit: It can be helpful to see all issue reports, as [`golangci-lint` does not show them all by default](https://golangci-lint.run/usage/configuration/#issues-configuration)

```yaml
issues:
  max-issues-per-linter: 0
  max-same-issues: 0
```